src/lib/modules/Relations/backoffice/components/RelationModal/RelationModal.js

```diff
@@ -70,7 +70,7 @@ export default class RelationModal extends Component {
         size="large"
         closeIcon
         trigger={
-          <>
+          <div>
```

Comment:
is this `div` needed for spacing?

Reply:
No, it was changed because otherwise we get this error now 
![image](https://user-images.githubusercontent.com/15194802/103779598-45881600-5034-11eb-9e57-b874481ce63d.png)